tests/util/test_rwlock.py

```diff
@@ -100,23 +100,54 @@ async def action():
         self.assertTrue(acquired_d.called)
         release_d.callback(None)
 
+    def _start_reader_or_writer(
+        self,
+        read_or_write: Callable[[str], AsyncContextManager],
+        key: str,
+        name: str,
+    ) -> Tuple["Deferred[None]", "Deferred[None]"]:
+        """Starts a reader or writer which acquires the lock, blocks, then completes."""
```

Comment:
some docs on the params and return vals would be helpful.

Reply:
Sounds like a good idea. Added some docs.